### ISSUE TITLE
Ballerina update command degrades to 0.8.0 when latest version is being used already

### ISSUE THREAD

**Issue by parkavi11** [ballerina-platform/ballerina-update-tool#19]
**Description:**
'ballerina update' command degrades to 0.8.0 when the active update tool version is already 0.8.4, instead of pointing that the latest version is already being used.

**Suggested Labels:**
Area/BuildTools

**Suggested Assignees:**

**Affected Product Version:**
jBallerina 1.2.0
Language specification 2020R1
Ballerina tool 0.8.4

**OS, DB, other environment details and versions:**    
CentOS 8

**Steps to reproduce:**
- Do ballerina -v to check the current version of the update-tool (tool version has to be the latest, in my case it is 0.8.4)
- Do ballerina update

**Related Issues:**

**Comment by suganyasuven** [comment]
Same issue occurs in Windows as well.
<img width="673" alt="updateissue" src="https://user-images.githubusercontent.com/30232477/76933478-33a57d00-6913-11ea-9b44-d34be7292f2f.png">

**Comment by irshadnilam** [comment]
Occurs on mac as well.

**Comment by Urahara** [comment]
It's looks like the server always send version **0.8.0** as the latest:

```sh
curl -k -X GET "https://api.central.ballerina.io/1.0/update-tool/versions/latest" -H "user-agent: jballerina/1.1.4 (linux-64) Updater/0.8.0" -H "accept: application/json" -H "host: api.central.ballerina.io" -H "connection: keep-alive"        
{"version":"0.8.0"}
```

I also can't update my tool either:
```sh
$ ballerina update 
Fetching the latest version from the remote server...
The latest tool version '0.8.0' is already in use
```

**Comment by Urahara** [comment]
This is fixed now on #22

**Comment by keizer619** [comment]
This is happened 0.8.5 was not published during RC period. Now this is fixed as we have published it
